Assets/MRTK/Providers/OpenXR/Scripts/MicrosoftArticulatedHand.cs

```diff
@@ -298,6 +298,9 @@ protected override void UpdatePoseData(MixedRealityInteractionMapping interactio
                             CoreServices.InputSystem?.RaisePoseInputChanged(InputSource, ControllerHandedness, interactionMapping.MixedRealityInputAction, interactionMapping.PoseData);
                         }
                         break;
+                    // IndexFinger is handled in ArticulatedHandDefinition, so we can safely skip this case.
```

Comment:
Don't think we need to break here. The GenericXRSDKController doesn't handle the IndexFinger at all, and I think it's probably not the best practice to prescribe things which we *don't* update the pose of, due to an implicit assumption that it is handled somewhere else. We should just focus on having this method handle the things it knows it can, and then leaving anything remaining to the base class, instead of overwriting the base classes behavior.

Reply:
You're right, this one is maybe a _bit_ of an over-optimization, but I think it's safe in this case. We know exactly what the base class handles, so there's no point in entering the base method and going through a second `switch` statement just to do nothing and return.

Reply:
I agree that it's a safe change, just a bit inelegant, especially because the ArticulatedHandDefinition isn't being referenced along the UpdatePoseData path, which makes this code a bit more confusing than it needs to be.